src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGANaivelySpark.java

```diff
@@ -0,0 +1,485 @@
+        filterArgs.add(prefix+".fa");
+        final Tuple3<Integer, String, String> filterInfo = filter.run(sgaPath, outputDirectory, filterArgs);
+
+        return new SGAUnitRuntimeInfoStruct("filer", filterInfo, prefix+".filter.pass.fa");
+    }
+
+    @VisibleForTesting
+    static SGAUnitRuntimeInfoStruct SGARmDuplicate(final Path sgaPath,
+                                                   final File inputFASTAFile,
+                                                   final File outputDirectory,
+                                                   final int threads,
+                                                   final SGAModule indexer,
+                                                   final ArrayList<String> indexerArgs){
+        return simpleModuleRun(sgaPath, "rmdup", ".rmdup.fa", inputFASTAFile, outputDirectory, threads, indexer, indexerArgs);
+    }
+
+    @VisibleForTesting
+    static SGAUnitRuntimeInfoStruct SGAFMMerge(final Path sgaPath,
+                                               final File inputFASTAFile,
+                                               final File outputDirectory,
+                                               final int threads,
+                                               final SGAModule indexer,
+                                               final ArrayList<String> indexerArgs){
+        return simpleModuleRun(sgaPath, "fm-merge", ".merged.fa", inputFASTAFile, outputDirectory, threads, indexer, indexerArgs);
+    }
+
+    @VisibleForTesting
+    static SGAUnitRuntimeInfoStruct SGAAssemble(final Path sgaPath,
+                                                final File inputFASTAFile,
+                                                final File outputDirectory,
+                                                final int threads){
+
+        final SGAModule overlap = new SGAModule("overlap");
+        final ArrayList<String> overlapArgs = new ArrayList<>();
+        overlapArgs.add("--threads"); overlapArgs.add(Integer.toString(threads));
+        overlapArgs.add(inputFASTAFile.getName());
+        final Tuple3<Integer, String, String> overlapInfo = overlap.run(sgaPath, outputDirectory, overlapArgs);
+
+        final String prefix = extractBaseNameWithoutExtension(inputFASTAFile);
+
+        final SGAModule assemble = new SGAModule("assemble");
+        final ArrayList<String> assembleArgs = new ArrayList<>();
+        assembleArgs.add("--out-prefix"); assembleArgs.add(prefix);
+        assembleArgs.add(prefix+".asqg.gz");
+        final Tuple3<Integer, String, String> assembleInfo = assemble.run(sgaPath, outputDirectory, assembleArgs);
+
+        final SGAUnitRuntimeInfoStruct assembleUnitInfo = new SGAUnitRuntimeInfoStruct("assemble", assembleInfo, prefix+"-contigs.fa");
+        assembleUnitInfo.add("overlap", overlapInfo);
+        return assembleUnitInfo;
+    }
+
+    // boiler plate code for running simple sga modules
```

Comment:
Could you add to this comment to describe very briefly what makes an SGA module 'simple'?